### PR TITLE
[PPML] add trusted-dl-serving image

### PR DESCRIPTION
**Pull request by gc-fu**
## Description
Add dl-serving image

## TODO
- [x] Test with nano_fp32_channels_last mar file
- [x]  Decide how we pass in the SGX_ENABLED, whether through a flag, or using environment variable?
## Probably in other prs
- [ ] Modify Entrypoint.sh after adding triton-server/tfserving etc.
- [ ] README.md

### CODE REVIEW

File: ppml/trusted-dl-serving/base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ENV TINI_VERSION                        $TINI_VERSION
+ARG JDK_VERSION
+ENV JDK_HOME                            /opt/jdk${JDK_VERSION}
+ENV JAVA_HOME                           /opt/jdk${JDK_VERSION}
+# Environment used for build pytorch
+RUN mkdir /ppml/examples && \
+    mkdir /ppml/torchserve
+
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
+ADD ./entrypoint.sh /opt/entrypoint.sh
```
Comment (gc-fu) [inline ppml/trusted-dl-serving/base/Dockerfile:38]: Shall we need an **entrypoint** for this image?
If yes, what should be included inside the entrypoint?

File: ppml/trusted-dl-serving/base/start-backend-sgx.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (glorysdj) [inline ppml/trusted-dl-serving/base/start-backend-sgx.sh:1]: rename to start-torchserv-backend-sgx.sh

File: ppml/trusted-dl-serving/base/start-frontend-sgx.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```
Comment (glorysdj) [inline ppml/trusted-dl-serving/base/start-frontend-sgx.sh:1]: rename to start-torchserve-frontend-sgx.sh

**Review by Hanyu-Jin — APPROVED** [review]

File: ppml/trusted-dl-serving/base/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+# COPY frontend.jar
+COPY --from=temp /ppml/torchserve/frontend.jar /ppml/torchserve/frontend.jar
+# Start Script for Torchserve
+ADD ./start-backend-sgx.sh      /ppml/torchserve/start-backend-sgx.sh
```
Comment (glorysdj) [inline ppml/trusted-dl-serving/base/Dockerfile:43]: name is not correct?

**Pull request merged**